public/data/cartData.json

```
@@ -0,0 +1,20 @@
+[
+  {
+    "cart_id": 1,
+    "item_id": 1,
+    "name": "[오리지널] 태스크 매니저 31DAYS - 화이트",
+    "price": "2000.00",
+    "discount": "1800.00",
```

```suggestion
    "price": 2000,
    "discount": 1800,
```
백엔드에서 꼭 소숫점 자리까지 포함한 문자열로 받아와야 하는건가요? 한번 확인해보시면 좋을거 같아요.

---

네 확인해서 수정했습니다!